src/ngraph/runtime/cpu/cpu_backend.cpp

```diff
@@ -80,8 +80,6 @@ bool runtime::cpu::CPU_Backend::call(shared_ptr<Function> func,
 {
     bool rc = true;
 
-    validate_call(func, outputs, inputs);
```

Comment:
Should we perhaps put this in a conditional compile or runtime conditional? It has a purpose and we may want it for debugging a model.

Reply:
this seems too low-level for a compilation flag. i was contemplating adding an optional argument to the call method. would you be ok with that?

Reply:
Perfect. thanks